### ISSUE TITLE
Script join_aug_pred.pl incompatible with option --uniqueGeneId

### ISSUE THREAD

**Issue by marchoeppner** [Gaius-Augustus/Augustus#272]
As the title states, the script for joining Augustus predictions across split jobs does not work if the option "--uniqueGeneId=true" is invoked. 

This is because of the regexp used to determine a gene start:

https://github.com/Gaius-Augustus/Augustus/blob/5a1c7baef970d0a27a32866168c25d05cd387ac7/scripts/join_aug_pred.pl#L103

I suppose the trailing 'g' could simply be ommitted to fix this. It's just a bit of an odd behavior that took me a few minutes to track down. I imagine others may run into this as well. 

Cheers,
Marc

**Comment by KatharinaHoff** [comment]
I agree, it's safe to ommitt the trailing g since we are matching the hash marks and the term start gene. That is true for the current Augustus. @MarioStanke  is it true older Augustus versions, too? 

I made a commit to fix this for current and old Augustus at https://github.com/Gaius-Augustus/Augustus/commit/af437758b8210355321d4d6362f3dec44355b392 but we might have to revert it for old Augustus versions if the string "### gene" is not unique for gene starts in the old gff output format.
